Detectors/MUON/MCH/Simulation/src/DEDigitizer.cxx

```diff
@@ -78,6 +84,24 @@ void DEDigitizer::processHit(const Hit& hit, const InteractionRecord& collisionT
   auto localX = mResponse.getAnod(lpos.X());
   auto localY = lpos.Y();
 
+  // calculate angle between track and wire assuming wire perpendicular to z-axis
+  auto thetawire = asin((lexit.Y() - lentrance.Y()) / hitlengthZ); // check sign convention between O2 and Aliroot
```

Comment:
shouldn't it be atan or atan2?
is the sign ok (hitlengthZ = lentrance.Z() - lexit.Z())?

Reply:
In aliroot, it looks like it is arcus sin: http://alidoc.cern.ch/AliRoot/master/_ali_m_u_o_nv1_8cxx_source.html line 428
However, I would think that atan2 is more sensible.

Reply:
> Is the z-axis in local coordinaties parallel to the z-axis within ALICE or is it rotated by 180 degrees?

Good question, I am not sure. Some DE are rotated but I don't know if that includes a rotation around x or y by 180°. One simple solution would be to compute the angle in the global coordinate system, neglecting the small inclination of some chambers w.r.t. to vertical (or it can be taken into account as it is fixed) and the little rotation of some DE around z from misalignments. This resemble to what was done in AliRoot by using the track momentum vector.

Reply:
Ok